src/main/java/com/keyboardbuilder/enterprise/dto/Build.java

```diff
@@ -2,20 +2,21 @@
 
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
-class Build {
+
+public class Build {
     @SerializedName("id")
-    private Integer id;
+    private int id;
     @SerializedName("caseType")
-    private String caseType;
+    private KeyboardCase caseType;
     @SerializedName("layout")
     private int layout;
     @SerializedName("keyCaps")
-    private String keyCaps;
+    private KeyCaps keyCaps;
     @SerializedName("pcb")
-    private String pcb;
+    private PCB pcb;
     @SerializedName("plate")
-    private String plate;
+    private Plate plate;
     @SerializedName("keySwitches")
-    private String keySwitches;
+    private KeySwitches keySwitches;
```

Comment:
1. I _might_ be misinterpreting this code here, but it doesn't make much sense to me to have DTO classes for each of these variables and to _not_ use them.

Reply:
I agree with your interpretation.
Overusing the String data type is often a sign of Object Oriented Design gone awry.  Using DTO types instead of Strings is better OOD.